meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0.bb

```diff
@@ -0,0 +1,15 @@
+HOMEPAGE = "https://github.com/barmalei/zebra"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
```

Comment:
SUMMARY and DESCRIPTION now *removed* from the patch, instead of being *fixed* according to review comments.